faker/providers/person/fr_QC/__init__.py

```diff
@@ -1,481 +1,3 @@
-from .. import Provider as PersonProvider
+from ..fr_CA import *
```

Comment:
I would rather have a subclass of the `fr_CA` that emits a warning on `__init__`:

```python3
import warnings

from ..fr_CA import Provider as FRCAProvider


class Provider(FRCAProvider):
    def __init__(self,, *args, **kwargs):
        warnings.warn(...)
        super().__init__(*args, **kwargs)

```